ming_sim/action_materialize.py

```
@@ -2007,6 +2007,260 @@ def validate_tingtui_appointment_shape(obj: Any) -> Tuple[bool, str]:
+    if mode_mark == "midzhi" and payload.get("mode") != "midzhi":
+        payload["mode"] = "midzhi"
+        changed = True
+    if tenure_mark == "署理" and payload.get("任别") != "署理":
+        payload["任别"] = "署理"
+        payload.pop("appointment_tenure", None)
+        changed = True
+    if not changed and (
+        (mode_mark == "midzhi" and payload.get("mode") == "midzhi")
+        or (tenure_mark == "署理" and payload.get("任别") == "署理")
+    ):
+        # 语义已在：仍回 id（no-op 去重存活），可补留痕
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)